spec/requests/admins/surveys_spec.rb

```diff
@@ -31,6 +31,12 @@
       post admin_study_surveys_path(study), params: { survey: attributes_for(:survey) }
       expect(response).to redirect_to(admin_study_survey_path(study, Survey.last))
     end
+
+    it "does not create a new survey end_date is before start_date" do
```

Comment:
I don't understand what this is testing. I would think based upon the use of the `invalid` trait and this text `does not create a new survey end_date is before start_date` that Survey.count wouldn't change. what am I misunderstanding?

Reply:
I think the issue is that when we create a study, it automatically creates a survey, so the initial Survey.count is set to 1. This is why the count changes in the test even when we expect it not to, so if we will create a new survey the with study the count will be 2 not 1